src/globals.cpp

```diff
@@ -209,6 +210,32 @@ CNodeSignals g_signals;
 CAddrMan addrman;
 CDoSManager dosMan;
 
+// Transaction mempool admission globals
+
+// Transactions that are available to be added to the mempool, and protection
+CCriticalSection csTxInQ;
+CCond cvTxInQ;
+
+// Finds transactions that may conflict with other pending transactions
+CFastFilter<4 * 1024 * 1024> incomingConflicts GUARDED_BY(csTxInQ);
+
+// Tranactions that are waiting for validation and are known not to conflict with others
+std::queue<CTxInputData> txInQ GUARDED_BY(csTxInQ);
```

Comment:
Would be useful to have a short write up of the work flow of how a transaction arrives and gets processed using the different Q's.

Reply:
I have a nice writeup IIRC.  I'll dig it up.